packages/pages/src/components/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./Image";
+export * from "./AspectRatio";
```

Comment:
Does this need to be exported to users? It looks like they only use Image.

Reply:
Only Image and AspectRatio should be exported, because those are dictated by the index.ts in the subdirectories.

I'm not sure if this is objectively cleaner but I do think one nice benefit is that in the future we could automate a check to verify that all the components have an export, and if they're in an easy format like `export * from "./FolderName"` then that's trivial to verify.

Reply:
Unexporting AspectRatio since it's basically just an internal component for Image